src/Fixer/ArrayNotation/NoTrailingCommaInSinglelineArrayFixer.php

```diff
@@ -84,7 +84,7 @@ private function fixArray(Tokens $tokens, $index)
 
         if ($beforeEndToken->equals(',')) {
             $tokens->removeTrailingWhitespace($beforeEndIndex);
-            $beforeEndToken->clear();
```

Comment:
`$beforeEndToken` variable is now used only once, so you can pass its value directly here instead.